teams/teams-ps/teams/Set-CsOnlineLisCivicAddress.md

```diff
@@ -33,6 +33,11 @@ Set-CsOnlineLisCivicAddress -CivicAddressId <Guid> [-CompanyName <String>] [-Com
 ## DESCRIPTION
 Use the `Set-CsOnlineLisCivicAddress` cmdlet to modify an existing civic address which has not been validated. Validated civic addresses cannot be modified.
```

Comment:
"which has not been validated. Validated civic addresses cannot be modified." is not valid anymore since we are allow users to edit the validated address too.